src/components/navigation/Navbar.tsx

```diff
@@ -1,12 +1,25 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import "./Navbar.scss";
 
 export const Navbar: React.FunctionComponent = () => {
   return <nav>
-    <ul>
-      <li><Link to="/">Home</Link></li>
-      <li><Link to="/rovers">Rovers</Link></li>
-      <li><Link to="/quiz">Quiz</Link></li>
-    </ul>
+
+    <div className="emptySpacingBox"></div>
+
+    <div id="menuToggle">
+      <input type="checkbox" />
+      
+        <span></span>
+        <span></span>
+        <span></span>
+      
+        <ul id="menu">
```

Comment:
Prefer using classes over IDs:
```suggestion
        <ul className="menu">
```

and also, "menu" is a little generic - there's this neat CSS class naming convention called [BEM](https://codeburst.io/understanding-css-bem-naming-convention-a8cca116d252) that has some good ideas. I think here we would say the block is the navbar, and the element is the list of links, so this class may be called `navbar__nav-links`!

Reply:
Done